components/viewController/viewController.js

```diff
@@ -3,22 +3,23 @@ app1.controller('viewController', function ($scope) {
 
 	$scope.views = [
 		{
-			jTitle: 'Import class list',
+			jTitle: 'Import Class List',
 			jDescription: 'Import the names of your students by pasting in a list or entering them using the form.'
 		},
 		{
-			jTitle: 'Create template',
+			jTitle: 'Create Template',
 			jDescription: 'Create a template that you will use to quickly generate reports based on the criteria you set.'
```

Comment:
tfw you read something right after pushing the rest of the changes :(
Want me to make another commit, or can this feature be delayed?